src/dbt_score/cli.py

```diff
@@ -93,8 +93,17 @@ def cli() -> None:
     is_flag=False,
     default=None,
 )
+@click.option(
+    "--show-all",
+    help="If set to True, show all models and all rules in output "
+    "when using `plain` as `--format`. "
+    "Default behavior is to only show failing models and violated rules.",
+    type=bool,
+    is_flag=True,
+    default=False,
+)
```

Comment:
Let's not make it a flag, but rather a parameter with a value? i.e. `--show all`, which can later be expanded into different options such as `--show failing-models` and the like